### PR TITLE
Nept 1180

### PR DESCRIPTION
**Pull request by imanoleguskiza**

### CODE REVIEW

File: templates/README.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 This folder is where you should place all your overriding template files. By
-default, the Bootstrap base theme provides all the necessary template files in
-various folders inside of sites/*/themes/bootstrap/theme. For example, the
+default, the Atomium base theme provides all the necessary template files in
+various folders inside of sites/*/themes/atomium/templates. For example, the
 page.tpl.php template file is located at
-sites/*/themes/bootstrap/theme/system/page.tpl.php. To override any of these
-files, copy them from the Bootstrap base theme and place them in here.
+sites/*/themes/atomium/templates/page/page.tpl.php. To override any of these
+files, copy them from the Atomium base theme and place them in here.
```
Comment (gillesdeudon) [inline templates/README.txt]: I suggest to put a reference like "For more information, please read the readme file of theatomium theme"

File: europa.info [hunk 2]
```diff
@@ -120,15 +84,6 @@ multisite_version = 2.2
...
 ; Necessary module CSS.
 
-;exclude[css][] = 'modules/block/block.css'
-;exclude[css][] = 'modules/node/node.css'
-;exclude[css][] = 'modules/system/admin.css'
-;exclude[css][] = 'modules/system/system.css'
-;exclude[css][] = 'modules/system/system.admin.css'
-;exclude[css][] = 'modules/system/system.base.css'
-;exclude[css][] = 'misc/vertical-tabs.css'
-;exclude[css][] = 'modules/filter/filter.css'
```
Comment (gillesdeudon) [inline europa.info]: Viewed with @imanoleguskiza and @Fefaine, I will reinject these exclusions.

File: templates/README.txt (hunk 1 above)
Comment (gillesdeudon) [inline templates/README.txt]: Viewed with @imanoleguskiza, I will put the reference

**Comment by gillesdeudon** [comment]
👍

**Comment by Fefaine** [comment]
👍

**Pull request merged**